lib/odometry-ng/OdometerDifferential.cpp

```diff
@@ -0,0 +1,44 @@
+#include <math.h>
+#include <stdlib.h>
+
+#include "odometer/OdometerDifferential.hpp"
+#include "trigonometry.h"
+#include "utils.hpp"
+
+namespace cogip {
+
+namespace odometer {
+
+int OdometerDifferential::update()
+{
+    if (!left_encoder_ && !right_encoder_) {
+        return -1;
+    }
+    
+    /// Compute encoders wheels left and right linear delta in mm
+    const double dL = left_encoder_->get_angle_and_reset() * parameters_.left_wheel_diameter_mm() * parameters_.left_polarity();
```

Comment:
left_encoder_ and right encoder_ should be tested as they can be nullptr

Reply:
Because we use reference, we can't have a null reference here. there is no need to check